design/clusters.go

```diff
@@ -155,6 +155,38 @@ var _ = a.Resource("clusters", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 	})
 
+	a.Action("findByURL", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.GET("/"),
+		)
+		a.Params(func() {
+			a.Param("cluster_url", d.String, "the URL of the cluster to show")
```

Comment:
I think we agreed about using `-` (~`cluster_url`~ -> `cluster-url`) previously for consistency across service

Reply:
thanks for the reminder, I used underscores as an habit :/
changed in e023199